app/javascript/packs/validators/validator_details.vue

```diff
@@ -0,0 +1,338 @@
+              </tr>
+
+              <tr>
+                <td><strong>IP:</strong></td>
+                <td>{{ score.ip_address }}</td>
+              </tr>
+
+              <tr>
+                <td><strong>Data Center:</strong></td>
+                <td>
+                  <a href="data_center_link(validator)" v-if="validator.dch_data_center_key">
+                    {{ validator.dch_data_center_key}}
+                  </a>
+                </td>
+              </tr>
+            </tbody>
+          </table>
+        </div>
+      </div>
+
+      <div class="col-lg-6 mb-4">
+        <div class="card h-100">
+          <div class="card-content pb-0">
+            <h2 class="h4 card-heading">Validator Details</h2>
+          </div>
+          <table class='table table-block-sm mb-0'>
+            <tbody>
+            <tr>
+              <td class="column-lg"><strong>Keybase:</strong></td>
+              <td>{{ validator.keybase_id }}</td>
+            </tr>
+
+            <tr>
+              <td><strong>Website:</strong></td>
+              <td><a :href="validator.www_url">{{ validator.www_url }}</a></td>
+            </tr>
+
+            <tr>
+              <td>
+                <strong>Vote Account:</strong>
+              </td>
+              <td>
+                <small class="word-break">
+                  <a href="vote_account_path(validator)" v-if="validator.vote_account_active">
+                    {{ validator.vote_account_active.account }}
+                  </a>
+                </small>
+              </td>
+            </tr>
+
+            <tr>
+              <td><strong>Active Stake:</strong></td>
+              <td>
+                {{ active_stake }} SOL
+              </td>
+            </tr>
+
+            <tr>
+              <td><strong>Commission:</strong></td>
+              <td :class="commission_class" data-turbolinks="false">
+                {{ score.commission }}&percnt;
+                <a :href="commission_histories_path(validator)" class="small">
+                  (See commission changes)
+                </a>
+              </td>
+            </tr>
+
+            <tr v-if="validator.security_report_url">
+              <td>
+                <strong>Security Info:</strong>
+              </td>
+              <td>
+                <a href="validator.security_report_url" target="_blank">
+                  validator.security_report_url
+                </a>
+              </td>
+            </tr>
+              <tr>
+                <td>
+                  <strong>Scores:</strong>
+                </td>
+                <td>
+                  <validator-scores :score="score"></validator-scores>
+                </td>
+              </tr>
+            </tbody>
+          </table>
+        </div>
+      </div>
+    </div>
+
+    <div class="row mb-3">
+      <div class="col-md-9 col-lg-10">
+        <a :href="'/validators?network=' + validator.network + '&order=' + order + '&refresh=' + refresh + '&page=' + page"
```

Comment:
There are a couple of params so I'd extract them to a separate variable, or even full path to a separate method.